### ISSUE TITLE
PDF files need a descriptive title and proper use of headings

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1453]
Our PDF files have the following accessibility issues:
- Missing descriptive titles
- Do not follow the proper `<h1><h2><h3>` hierarchy (e.g. start with `<h2>` just because the font size is preferred)

For information on descriptive PDF titles: https://www.w3.org/TR/WCAG20-TECHS/PDF18.html

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'Medium' priority.

**Comment by pherterich** [comment]
@martaribeiro - something you could try and address while tackling the accessibility issues for DMPonline?

**Comment by gjacob24** [comment]
Closing this issue as it will be tracked in https://github.com/DMPRoadmap/roadmap/issues/3452
